### PR TITLE
Adds possibility to specify batch_norm, bias and batch_norm_scale for each layer

### PR DESCRIPTION
**Pull request by MaxFBurg**
To specify batch norm and its properties for each layer individually, this PR changes the attributes `batch_norm` `bias` `batch_norm_scale` and the according class members to lists. 

Backwards-compatibility is ensured.

Also includes #215 which should be merged first.

### CODE REVIEW

**Comment by KonstantinWilleke** [comment]
Thanks a lot Max! I've built that branch with the sensorium model and tried it out - worked as intended.

**Comment by KonstantinWilleke** [comment]
The branch is out of date with the base branch though - could you update it? then I'll go ahead and merge

**Comment by MaxFBurg** [comment]
Thank you @KonstantinWilleke - merged the latest changes from main, so this PR should now be ready for merging into `main`.

**Comment by MaxFBurg** [comment]
Will close #215 (included in this PR)
